src/main/java/org/example/HomeWork1.java

```diff
@@ -0,0 +1,120 @@
+package main.java.org.example;
+
+import java.util.*;
+import java.util.stream.IntStream;
+
+public class HomeWork1 {
+
+    //Problem 1...
+    public int getMaxWordOccurrence(String text){
+
+        List<String> sentences;
+        List<Integer> sentenceLengths = new ArrayList<>();
+        //Here we are converting text to array of sentences, we are replacing . ? ! with _ symbol and then
+        //splitting this text into arrays with _ symbol.
```

Comment:
You don`t have to comment every single operation you do. Remember, if you have to comment your code process, than you might be able to improve it.